styleguide/source/_patterns/01-molecules/06-components/11-hub-card/_hub-card.scss

```diff
@@ -42,3 +42,31 @@
   margin: 0 24px;
   order: 2;
 }
+
+.hub-card-portrait {
+  min-height: 300px;
```

Comment:
Also, I can set up the mobile one in this branch, but I'll have to do the height variations in the row branch since that is dependent on children.